lib/index.js

```diff
@@ -51,8 +57,8 @@ class BaseAsset {
 
     get asset_id() { return this._asset_id; }
 
-    set_title(value) { this._title = value; }
-    set_synopsis(value) { this._synopsis = value; }
+    set_title(value) { this._title = _sanitize_text(value); }
```

Comment:
great, I'd like to keep the _sanitize_text function though, in case we want to add more things later to the sanitation. Any strong opinions? @ptomato @wgreenberg